.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Does this affect Java files too? Currently handwritten Java files in this repo use 4 spaces indentation.